### DISCUSSION TITLE
Can not perform pull due to some errors

### DISCUSSION THREAD

**Discussion by wiatrgr** [dani-garcia/vaultwarden#3854] [Q&A]
Hello, has anyone experienced issues with updating Vaultwarden and got errors like these? 

`vaultwarden]# docker pull vaultwarden/server:latest`

`Error response from daemon: error unmarshalling content: invalid character '<' looking for beginning of value` ?

I was trying to update from version 1.29.1 to version 1.29.2, and this is the first time I had seen this problem.  Logs show no issues at all.

**Comment by BlackDex** [comment]
Which version of docker are you using? `docker info` should provide that information.

**Comment by wiatrgr** [comment]
Docker version 20.10.17, build 100c701

**Reply by BlackDex** [reply]
Please update docker as described in this discussion #3827
